python.d/go_expvar.chart.py

```diff
@@ -0,0 +1,229 @@
+# -*- coding: utf-8 -*-
+# Description: go_expvar netdata python.d module
+# Author: Jan Kral (kralewitz)
+
+from __future__ import print_function
```

Comment:
Fixed in [b0e40cb9](https://github.com/firehol/netdata/pull/2280/commits/b0e40cb90655a92f9c9b54888fa1f82316bf5b09)